rfd/0080/convergence-service-draft.md

```diff
@@ -0,0 +1,49 @@
+# Mariposa Convergence Service
+
+The Mariposa (RFD36) convergence service will be responsible for insuring service goal state is met. It accomplishes this by polling the service and project APIs, the VM API, and by monitoring Changefeed. When a divergence between goal state and actual state is detected, the convergence service will interact with VM API to resolve the discrepancy. 
```

Comment:
I should add that I'm not saying this document needs to present a solution to the problem _now_, but we should identify it as a risk that demands attention.